Lib/test/test_math.py

```diff
@@ -731,7 +743,7 @@ def testGcd(self):
         self.assertEqual(gcd(-b, -a), c)
 
         self.assertRaises(TypeError, gcd, 120.0, 84)
-        self.assertRaises(TypeError, gcd, 120, 84.0)
+        self.assertRaises(TypeError, gcd, 120.0, 84, 23.5, 34.0, 78.0)
```

Comment:
Existing tests are sacred, so I think new tests should be independently added, not tweaked from existing tests.